specs/Standards/Sould Bound Token/README.md

```diff
@@ -0,0 +1,450 @@
+---
+NEP: 393
+Title: Soulbound Token
+Authors: Robert Zaremba <@robert-zaremba>
+DiscussionsTo:
+Status: Draft
+Type: Standards Track
+Category: Contract
+Created: 12-Sep-2022
+Requires:
+---
+
+# NEP: Soulbound Token
+
+## Summary
+
+Soulbound Token (SBT) is a form of a NFT which represents an aspect of an account: _soul_. Transferability is limited only to a case of recoverability or transferring the soul - a _soul transfer_ should coordinate with a registry to transfer all SBTs from one account to another, and _killing_ the source account.
+
+SBTs are well suited for carrying proof-of-attendance, proof-of-unique-human "stamps" and other similar credibility-carriers.
+
+## Motivation
+
+Recent [Decentralized Society](https://www.bankless.com/decentralized-society-desoc-explained) trends open a new area of Web3 research to model various aspects of what characterizes humans. Economic and governance value is generated by humans and their relationship. SBTs can represent the commitments, credentials, and affiliations of “Souls” that encode the trust networks of the real economy to establish provenance and reputation.
+
+> More importantly, SBTs enable other applications of increasing ambition, such as community wallet recovery, sybil-resistant governance, mechanisms for decentralization, and novel markets with decomposable, shared rights. We call this richer, pluralistic ecosystem “Decentralized Society” (DeSoc)—a co-determined sociality, where Souls and communities come together bottom-up, as emergent properties of each other to co-create plural network goods and intelligences, at a range of scales.
+
+Creating a strong primitives is necessary to model new innovative systems and decentralized societies. Examples include reputation protocols, undercollaterized lending, one-person-one-vote, fair airdrops & ICOs, universal basic income, non KYC identity systems, Human DAOs and methods for Sybil attack resistance.
+
+We propose a SBT standard to model protocols described above.
+
+_Verifiable Credentials_ (VC) could be seen as subset of SBT. However there is an important distinction: VC require set of claims and privacy protocols. It would make more sense to model VC with relation to W3 DID standard. SBT is different, it doesn't require a [resolver](https://www.w3.org/TR/did-core/#dfn-did-resolvers) nor [method](https://www.w3.org/TR/did-core/#dfn-did-methods) registry. For SBT, we need something more elastic than VC.
+
+## Specification
+
+Main requirement for Soulbound tokens is to bound an account to a human. A **Soul** is an account with SBTs, which are used to define account identity. Moving SBTs from one account to another should be strictly limited to:
```

Comment:
Just a little suggestion: "A Soul is an account with SBTs, which are used to define account identity." - it seems a very important sentence... maybe it would be cool to see it in a separate paragraph.

Reply:
In typesetting we don't put every sentence in it's own paragraph. It's at the beginning and it's emphasized.